### ISSUE TITLE
lincom missing plus sign in output

### ISSUE THREAD

**Issue by adw96** [statdivlab/rigr#92]
```
mod_rigr <- regress("mean", atrophy ~ age + sex, data = mri)
mod_combo <- c(0, -10, 1)
lincom(mod_rigr, mod_combo)
```

has `H0: -10*age1*sexMale   =  0` in its output. Could it please be `H0: -10*age+1*sexMale   =  0` instead? (i.e., the plus sign is missing)

Thank you!

**Comment by cwolock** [comment]
Yeah, I just noticed a similar bug in how output is being printed...thought I sorted this out but apparently not! Working on it now.

**Comment by cwolock** [comment]
Should be fixed by 7ba3268dcf789112483148b1f8e84062f0cb248f. @adw96 could you let me know how it looks to you?

**Comment by adw96** [comment]
looks great! thanks @cwolock !
